packages/playground/src/components/select_node.vue

```diff
@@ -172,6 +188,14 @@ watch(
   () => selectedNode.value,
   async node => {
     errorMessage.value = ``;
+    const grid = await getGrid(profileManager.profile!);
+    if (node && grid) {
+      if (props.selection === "automated") {
```

Comment:
We can use the selection enum here instead.
```suggestion
      if (props.selection === Selection.AUTOMATED) {
```